frontend/src/components/TextEditor.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+// starter code from https://www.npmjs.com/package/react-quilljs
+import { useQuill } from "react-quilljs";
+
+import "quill/dist/quill.snow.css"; // Add css for snow theme
+
+const TextEditor = () => {
+  const theme = "snow";
+
+  const modules = {
+    toolbar: [["bold", "italic", "underline", "strike"]],
+  };
+
+  const placeholder = "Add notes on the doc!";
```

Comment:
What's the purpose of this text editor? Are you going to save notes in the data base?